plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotGrpcServerQueryClientConfig.java

```diff
@@ -72,4 +74,28 @@ public PinotGrpcServerQueryClientConfig setUsePlainText(boolean usePlainText)
         this.usePlainText = usePlainText;
         return this;
     }
+
+    public String getProxyHost()
+    {
+        return proxyHost;
+    }
+
+    @Config("pinot.grpc.proxy-host")
+    public PinotGrpcServerQueryClientConfig setProxyHost(String proxyHost)
+    {
+        this.proxyHost = proxyHost;
+        return this;
+    }
+
+    public int getProxyPort()
+    {
+        return proxyPort;
+    }
+
+    @Config("pinot.grpc.proxy-port")
+    public PinotGrpcServerQueryClientConfig setProxyPort(int proxyPort)
+    {
+        this.proxyPort = proxyPort;
+        return this;
+    }
```

Comment:
If you expect both property should be set and don't expect setting only either one of the properties, please add assertion likes `MySqlJdbcConfig`.